### PR TITLE
add slot_padding_zero, test=develop

### PR DESCRIPTION
**Pull request by ForFishes**
PR types: Function optimization  

PR changes: Others

Describe: Add options for set_slot_padding_zero, its default is true. When it set false, it will rm 0 slot.

### CODE REVIEW

File: paddle/fluid/framework/data_feed.cc [hunk 1]
```diff
@@ -357,6 +358,11 @@ void InMemoryDataFeed<T>::SetCurrentPhase(int current_phase) {
   current_phase_ = current_phase;
 }
 
+template <typename T>
+void InMemoryDataFeed<T>::SetSlotPaddingZero(bool slot_padding_zero) {
```
Comment (hutuxian) [inline paddle/fluid/framework/data_feed.cc]: where to call this function?

File: paddle/fluid/operators/sequence_ops/sequence_pool_op.h [hunk 2]
```diff
@@ -43,10 +43,6 @@ class SequencePoolKernel : public framework::OpKernel<T> {
                                         "does not contain LoD information."));
     PADDLE_ENFORCE_LE(lod_level, 2UL,
                       "The lod level of input shall be no more than 2.");
-    PADDLE_ENFORCE_GE(
```
Comment (hutuxian) [inline paddle/fluid/operators/sequence_ops/sequence_pool_op.h:46]: need confirmation of this op's owner

File: paddle/fluid/framework/data_feed.cc (hunk 1 above)
Comment (ForFishes) [reply paddle/fluid/framework/data_feed.cc]: i will delete it.

File: paddle/fluid/operators/sequence_ops/sequence_pool_op.h (hunk 2 above)
Comment (ForFishes) [reply paddle/fluid/operators/sequence_ops/sequence_pool_op.h:46]: Ok. Done.

**Pull request closed without merging**